### ISSUE TITLE
Add option for additionalProperties keys to not be changed in the display

### ISSUE THREAD

**Issue by alexgmin** [bhch/django-jsonform#115]
Using the same example from https://github.com/bhch/django-jsonform/issues/114

Schema
```
{
    'type': 'dict',
    'title': 'Language ISO 2 letter code',
    'properties': {},
    'additionalProperties': {
        'type': 'dict',
        'title': 'Translation',
        'properties': {},
        'additionalProperties': {
            'type': 'string',
        },
    },
}
```

Display
![image](https://github.com/bhch/django-jsonform/assets/8340441/067452c8-2913-4cc2-8eaf-042703b1115e)

In this case, "Original name" internally is `original_name`
![image](https://github.com/bhch/django-jsonform/assets/8340441/56f9f85a-6774-4007-9583-c4cfd1e1c342)

Ideally there should be an option for the display of the key to not be "beautified" for cases like this. In this example the ISO language code is also lowercase, so ideally that would also be shown lowercase in this case.

**Comment by bhch** [comment]
Yeah, I had also been a bit unsure about this when I first added this "feature".

Displaying the key names as they are seems a better approach.

I'll try to release a fix in a couple of days.

**Comment by bhch** [comment]
Fixed in [v2.18.0](https://github.com/bhch/django-jsonform/releases/tag/v2.18.0). Please upgrade and test it out.

Thank you for reporting the issue.
